RotaryWheelControl/RotaryWheel.xaml.cs

```diff
@@ -25,7 +25,9 @@ public partial class RotaryWheel : UserControl, INotifyPropertyChanged
         public Color BackgroundColor
         {
             get { return _backgroundColor; }
-            set { SetField(ref _backgroundColor, value); }
+            set { SetField(ref _backgroundColor, value); 
+            Draw();
```

Comment:
Yes, i've experienced that without Draw() function call, BackgroundColor doesnot changes instantly